example/package.json

```diff
@@ -10,9 +10,9 @@
     "@types/node": "^16.11.44",
     "@types/react": "^18.0.15",
     "@types/react-dom": "^18.0.6",
-    "ra-data-odata-server": "file:../dist",
+    "@4efficiency/ra-data-odata-server": "file:../",
```

Comment:
Is this change just for testing your fork? I think you'll need to keep that downstream.

Reply:
Well... since i really had to move on for a project, we decided to create a forked version. "@4efficiency/ra-data-odata-server".

That change was just for testing indeed!

Just fixed the test again and removed @4efficiency namespace.